modules/navier_stokes/src/fvkernels/INSFVMomentumAdvection.C

```diff
@@ -41,6 +42,15 @@ INSFVMomentumAdvection::validParams()
   params += INSFVMomentumAdvection::uniqueParams();
   params.addRequiredParam<MooseFunctorName>(NS::density, "Density functor");
   params.addClassDescription("Object for advecting momentum, e.g. rho*u");
+
+  params.addParam<bool>("momentum_norm_limiter",
```

Comment:
This sounds like it's asking for a type of limiter. Maybe `use_norm_for_momentum_limiter`?

Reply:
changed